packages/next-swc/crates/next-core/src/next_app/metadata/route.rs

```diff
@@ -0,0 +1,364 @@
+//! Rust port of the `next-metadata-route-loader`
```

Comment:
Can we extract the file templates out to the `next.js/packages/next/src/build/templates/` directory?

Reply:
I tried for a couple of hours, but there are problems with imports and rewriting the imported names in the image metadata loader doesn't work at all